packages/contracts/contracts/StabilityPool.sol

```diff
@@ -259,7 +259,7 @@ contract StabilityPool is LiquityBase, StabilityPoolStorage, CheckContract, ISta
 
         ICommunityIssuance communityIssuanceCached = communityIssuance;
 
-        _triggerZEROIssuance(communityIssuanceCached);
+        _triggerSOVIssuance(communityIssuanceCached, _amount);
```

Comment:
All calls to `_triggerSOVIssuance` are incorrect, because they pass only the amount added to the pool instead of the total ZUSD balance of the pool. the SOV issuance always only depends on the total ZUSD balance of the pool. you can always pass `totalZUSDDeposits`
So I suggest to remove the argument from `_triggerSOVIssuance` and update the call to `issueSOV` to pass `totalZUSDDeposits`